test/realizations/catchments/Bmi_Cpp_Formulation_Test.cpp

```diff
@@ -335,7 +335,7 @@ TEST_F(Bmi_Cpp_Formulation_Test, GetOutputLineForTimestep_1_b) {
         formulation.get_response(i++, 3600);
     formulation.get_response(i, 3600);
     std::string output = formulation.get_output_line_for_timestep(i, ",");
-    ASSERT_EQ(output, "0.000000,0.000002");
+    ASSERT_EQ(output, "0.000000,0.000001");
```

Comment:
I'll be kind of surprised if the `-0` thing only happens on C and not C++ ... but we can keep an eye on it.